because user namespaces have been disabled
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Fri Apr  5 21:37:10 2024... 
[Fri Apr  5 21:52:01 2024]  Iteration number: 0 with current cost as 0.33435917266295423 and parameters 
[-1.49277898  2.23743464 -2.12427981 -0.11653068  0.55388748 -2.77010812
  3.06858487  2.18960174  1.18551976 -1.06648337  0.6027151   1.14432439
  1.31029933 -1.87354652  0.7296508   2.88578431 -0.54534295 -0.47522468
 -2.02654195  0.72897341  1.60512721  2.83077124 -1.2645671  -0.2513611
 -2.39279201 -2.27309809  3.13337138  2.54856975 -0.6755077  -2.69002139
  1.5170164  -0.04647234 -2.34271399  2.04043168  1.15216369 -0.6505543
  1.04465352 -1.92056021 -0.32067971  2.32226067 -0.20585098  0.56637433
  2.68983137  1.07273396  2.58102924  2.94254089  2.76514556 -2.94578605
  0.3668415  -0.65725918  1.65159149  2.68042147 -0.2519492  -2.45807895
  2.6357053  -3.07930429  0.79129724 -0.75901765 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Mon Apr  8 15:10:59 2024... 
[Mon Apr  8 15:25:46 2024]  Iteration number: 0 with current cost as 0.33435917266295423 and parameters 
[-1.49277898  2.23743464 -2.12427981 -0.11653068  0.55388748 -2.77010812
  3.06858487  2.18960174  1.18551976 -1.06648337  0.6027151   1.14432439
  1.31029933 -1.87354652  0.7296508   2.88578431 -0.54534295 -0.47522468
 -2.02654195  0.72897341  1.60512721  2.83077124 -1.2645671  -0.2513611
 -2.39279201 -2.27309809  3.13337138  2.54856975 -0.6755077  -2.69002139
  1.5170164  -0.04647234 -2.34271399  2.04043168  1.15216369 -0.6505543
  1.04465352 -1.92056021 -0.32067971  2.32226067 -0.20585098  0.56637433
  2.68983137  1.07273396  2.58102924  2.94254089  2.76514556 -2.94578605
  0.3668415  -0.65725918  1.65159149  2.68042147 -0.2519492  -2.45807895
  2.6357053  -3.07930429  0.79129724 -0.75901765 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Mon Apr  8 16:22:29 2024... 
[Mon Apr  8 16:37:59 2024]  Iteration number: 0 with current cost as 0.33435917266295423 and parameters 
[-1.49277898  2.23743464 -2.12427981 -0.11653068  0.55388748 -2.77010812
  3.06858487  2.18960174  1.18551976 -1.06648337  0.6027151   1.14432439
  1.31029933 -1.87354652  0.7296508   2.88578431 -0.54534295 -0.47522468
 -2.02654195  0.72897341  1.60512721  2.83077124 -1.2645671  -0.2513611
 -2.39279201 -2.27309809  3.13337138  2.54856975 -0.6755077  -2.69002139
  1.5170164  -0.04647234 -2.34271399  2.04043168  1.15216369 -0.6505543
  1.04465352 -1.92056021 -0.32067971  2.32226067 -0.20585098  0.56637433
  2.68983137  1.07273396  2.58102924  2.94254089  2.76514556 -2.94578605
  0.3668415  -0.65725918  1.65159149  2.68042147 -0.2519492  -2.45807895
  2.6357053  -3.07930429  0.79129724 -0.75901765 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 330, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Mon Apr  8 19:23:17 2024... 
[Mon Apr  8 19:40:01 2024]  Iteration number: 0 with current cost as 0.33435917266295423 and parameters 
[-1.49277898  2.23743464 -2.12427981 -0.11653068  0.55388748 -2.77010812
  3.06858487  2.18960174  1.18551976 -1.06648337  0.6027151   1.14432439
  1.31029933 -1.87354652  0.7296508   2.88578431 -0.54534295 -0.47522468
 -2.02654195  0.72897341  1.60512721  2.83077124 -1.2645671  -0.2513611
 -2.39279201 -2.27309809  3.13337138  2.54856975 -0.6755077  -2.69002139
  1.5170164  -0.04647234 -2.34271399  2.04043168  1.15216369 -0.6505543
  1.04465352 -1.92056021 -0.32067971  2.32226067 -0.20585098  0.56637433
  2.68983137  1.07273396  2.58102924  2.94254089  2.76514556 -2.94578605
  0.3668415  -0.65725918  1.65159149  2.68042147 -0.2519492  -2.45807895
  2.6357053  -3.07930429  0.79129724 -0.75901765 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Wed Apr 10 06:44:14 2024... 
[Wed Apr 10 07:02:34 2024]  Iteration number: 0 with current cost as 0.33435917266295423 and parameters 
[-1.49277898  2.23743464 -2.12427981 -0.11653068  0.55388748 -2.77010812
  3.06858487  2.18960174  1.18551976 -1.06648337  0.6027151   1.14432439
  1.31029933 -1.87354652  0.7296508   2.88578431 -0.54534295 -0.47522468
 -2.02654195  0.72897341  1.60512721  2.83077124 -1.2645671  -0.2513611
 -2.39279201 -2.27309809  3.13337138  2.54856975 -0.6755077  -2.69002139
  1.5170164  -0.04647234 -2.34271399  2.04043168  1.15216369 -0.6505543
  1.04465352 -1.92056021 -0.32067971  2.32226067 -0.20585098  0.56637433
  2.68983137  1.07273396  2.58102924  2.94254089  2.76514556 -2.94578605
  0.3668415  -0.65725918  1.65159149  2.68042147 -0.2519492  -2.45807895
  2.6357053  -3.07930429  0.79129724 -0.75901765 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Wed Apr 10 17:34:40 2024... 
[Wed Apr 10 17:49:01 2024]  Iteration number: 0 with current cost as 0.33435917266295423 and parameters 
[-1.49277898  2.23743464 -2.12427981 -0.11653068  0.55388748 -2.77010812
  3.06858487  2.18960174  1.18551976 -1.06648337  0.6027151   1.14432439
  1.31029933 -1.87354652  0.7296508   2.88578431 -0.54534295 -0.47522468
 -2.02654195  0.72897341  1.60512721  2.83077124 -1.2645671  -0.2513611
 -2.39279201 -2.27309809  3.13337138  2.54856975 -0.6755077  -2.69002139
  1.5170164  -0.04647234 -2.34271399  2.04043168  1.15216369 -0.6505543
  1.04465352 -1.92056021 -0.32067971  2.32226067 -0.20585098  0.56637433
  2.68983137  1.07273396  2.58102924  2.94254089  2.76514556 -2.94578605
  0.3668415  -0.65725918  1.65159149  2.68042147 -0.2519492  -2.45807895
  2.6357053  -3.07930429  0.79129724 -0.75901765 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Traceback (most recent call last):
  File "/home/gjones/scratch/lin16qubit/main.py", line 329, in <module>
    main()
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/miniconda/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/gjones/scratch/lin16qubit/main.py", line 189, in main
    model.fit(X_train, y_train, load_state=resume_file)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 321, in fit
    self._save_partial_state(params, force=True)
  File "/home/gjones/scratch/lin16qubit/quantum/Quantum.py", line 237, in _save_partial_state
    os.remove('partial_state_model.bin')
FileNotFoundError: [Errno 2] No such file or directory: 'partial_state_model.bin'
/home/gjones/scratch/lin16qubit/quantum/Quantum.py:302: OptimizeWarning: Unknown solver options: tol
  opt_result = minimize(self._cost_wrapper, x0=params, method=self.optimizer, callback=self._callback,
Loading dataset from /linear_train.bin... 
Successfully loaded /linear_train.bin into X and y data. 
Loading dataset from /linear_test.bin... 
Successfully loaded /linear_test.bin into X and y data. 
Training model with dataset /linear_train.bin 
 at time Wed Apr 10 19:25:03 2024... 
[Wed Apr 10 19:40:17 2024]  Iteration number: 0 with current cost as 0.33435917266295423 and parameters 
[-1.49277898  2.23743464 -2.12427981 -0.11653068  0.55388748 -2.77010812
  3.06858487  2.18960174  1.18551976 -1.06648337  0.6027151   1.14432439
  1.31029933 -1.87354652  0.7296508   2.88578431 -0.54534295 -0.47522468
 -2.02654195  0.72897341  1.60512721  2.83077124 -1.2645671  -0.2513611
 -2.39279201 -2.27309809  3.13337138  2.54856975 -0.6755077  -2.69002139
  1.5170164  -0.04647234 -2.34271399  2.04043168  1.15216369 -0.6505543
  1.04465352 -1.92056021 -0.32067971  2.32226067 -0.20585098  0.56637433
  2.68983137  1.07273396  2.58102924  2.94254089  2.76514556 -2.94578605
  0.3668415  -0.65725918  1.65159149  2.68042147 -0.2519492  -2.45807895
  2.6357053  -3.07930429  0.79129724 -0.75901765 -1.81776562  2.9986886
 -1.85887169 -1.63734088 -2.55370824  2.30926632 -2.25921323  1.98533552
  1.90881014  1.42517315  2.59930885  2.87147469  1.84732041  1.52137428
 -1.69866533 -2.34431217  1.23292411]. 
Training complete taking 1599.5947840213776 total seconds. 
Now scoring model... 
Scoring complete taking 2.1455790996551514 seconds. 
Saved predicted values as A1-A1-CZ_Modified-Pauli-CRZ_predicted_values.csv
Model scores: {'MSE_train': (206.93809018355296,), 'R2_train': -0.0025455952221153932, 'MAE_train': 12.577383505470042, 'MSE_test': 191.4293743974114, 'R2_test': -0.1539006457830192, 'MAE_test': 12.012863818275061}. 
Saved model results as A1-A1-CZ_Modified-Pauli-CRZ_results.json. 
